tool/release/createnotes/CreateNotes.kt

```diff
@@ -50,6 +51,6 @@ private fun writeNotesMd(notes: List<Note>, releaseTemplateFile: Path) {
     val template = releaseTemplateFile.toFile().readText()
     if (!template.matches(".*${Constant.releaseTemplateRegex.pattern}.*".toRegex(RegexOption.DOT_MATCHES_ALL)))
         throw RuntimeException("The release-template does not contain the '${Constant.releaseTemplateRegex}' placeholder")
-    val markdown = template.replace(Constant.releaseTemplateRegex, Note.toMarkdown(notes))
+    val markdown = template.replace(Constant.releaseTemplateRegex, escapeReplacement(Note.toMarkdown(notes)))
```

Comment:
the `replace(regex, string)` actually treats the replacement string as a sort of regex itself, so we escape it to get the literal string value into the template